src/components/image-card-grid/image-card-grid.vue

```diff
@@ -24,25 +27,46 @@
         <structured-text-block
           :content="card.body"
         />
+
+        <app-button
+          v-if="card.link?.url"
+          class="image-card-grid__link"
+          :label="card.link.title"
+          :to="card.link.url"
+          external
+          secondary
+        />
+        <app-button
+          v-if="card.link?.page"
+          class="image-card-grid__link"
+          :label="card.link.title"
+          :to="createHref($i18n, card.link)"
+          secondary
+        />
```

Comment:
Two thoughts:
- Instead of checking on properties `url` & `page`, what do you think of testing against `__typename`? We do this in the structured text block too to see if a URL is external.
- Can we get rid of using createHref? Looks like we only need it to link to a service page, if we don't allow that for now but need to link to it we can use an external link. Then we can just pass an object with a slug.

Reply:
- `__typename` is not available on the link model: https://voorhoede-website.admin.datocms.com/admin/item_types/55657. That's only available in the link block. I'll explain the difference in the thread of your other comment.
- Jasper and Vera actually really would like to link to other page types at the moment. External links are suboptimal IMO since they open in a new tab. I think we would offer the best content management experience by allowing other pages at the moment, while getting rid of the pages we want to replace with the slug pages.

Reply:
:+1: linking to service pages is necessary evil then. Once we change service slugs to include services/ or change them to landing pages we can get rid of createHref.